doc/Web_Test_Suite_Developer_Guide.md

```diff
@@ -380,135 +380,9 @@ Note:
     </script>
 
 
-## Appendix 1 Tests.full.xml and tests.xml
```

Comment:
This `doc/Web_Test_Suite_Developer_Guide.md` documentation looks more like for the WebAPI part only. If no objection, I will rename it to `doc/WebAPI_Test_Suite_Developer_Guide.md` in a separate pull request.